bitnami/pinniped/Chart.yaml

```diff
@@ -22,4 +22,4 @@ name: pinniped
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/pinniped
   - https://github.com/vmware-tanzu/pinniped/
-version: 0.4.6
\ No newline at end of file
+version: 0.4.7
```

Comment:
Since this is adding a feature, please increase the minor.

Reply:
I've updated 1.0.0 since it's definitely a breaking change and it fixes a bux. See 54eee0ea2acb33dfffed531e3d7fb05b92d704b6.